cli/cmd/volume.go

```diff
@@ -73,6 +74,7 @@ func init() {
 	volumeCmd.AddCommand(volumeInfoCmd)
 	volumeCmd.AddCommand(volumeStatusCmd)
 	volumeCmd.AddCommand(volumeListCmd)
+	volumeCmd.AddCommand(volumeExpandCmd)
```

Comment:
Before this line, Flag need to be defined

	volumeExpandCmd.Flags().IntVarP(&flagCreateCmdReplicaCount, "replica", "", 0, "Replica Count")